boards/arm/cxd56xx/common/src/cxd56_altmdm.c

```diff
@@ -313,6 +316,12 @@ static bool altmdm_sready(void)
 
 static void altmdm_master_request(bool request)
 {
+  /* If the GPIO falls within 300us after raising
+   * (or GPIO raises within 300us after falling), the modem may miss the GPIO
+   * interrupt. So delay by 300us before changing the GPIO.
+   */
+
+  up_udelay(WAIT_READY_TO_GPIO_INTERRUPT);
```

Comment:
Is this in the running context or just for initialization?  How will this effect the real time performance?

Reply:
Yes, this is called when communicating with the modem. It occupies the CPU processing, and so it will affect the real-time performance as you said. This is a workaround for the communication with the modem.
We can use the OS usleep() function to run other processes, but the performance of modem communication will obviously decrease due to the delay of OS tick. As an overall performance, we choose a little time up_udelay() here.

Reply:
@davids5 @jerpelea I only have this solution at the moment. I would like to consider an implementation without udelay in the future. How about it?

Reply:
fine by me 
@davids5 what do you think?